src/main/java/com/mongodb/jdbc/oidc/manualtests/TestOidcCallbackWithShortTimeout.java

```diff
@@ -38,8 +39,6 @@ public static void main(String[] args) {
             System.out.println(
                     "This should not print, timeout expected. Sign out of the IdP to trigger a timeout.");
             System.out.println(result);
-        } catch (OidcTimeoutException e) {
```

Comment:
Here too, you want to catch runtime exception and check the cause to verify that it is the excepted reason for the failure.

Reply:
Same here, added the check for `instanceof OidcTimeoutException`